tktermwidget/tkterm.py

```diff
@@ -37,18 +40,20 @@ def __init__(self, master=None, **kwargs):
         Scrollbar.__init__(self, master=master, **kwargs)
 
     def set(self, first: int, last: int):
+        """Set the Scrollbar"""
         if float(first) <= 0.0 and float(last) >= 1.0:
             self.grid_remove()
         else:
             self.grid()
```

Comment:
Could we make this more explicitly so the horizontal scrollbar at never goes beneath the vertical scrollbar?

Reply:
What?

Reply:
![image](https://github.com/littlewhitecloud/TkTerminal/assets/71159641/15e7bc0c-bddb-41e8-bae2-dd41e548bf99)
??????

Reply:
> Could we make this more explicitly so the horizontal scrollbar at never goes beneath the vertical scrollbar?

What do you mean>

Reply:
LOL ignore it in this prl.

Reply:
.grid(row=0 if horizontal else 1, column=0 if horizontal else 1)

Reply:
@Moosems would like to make a suggestion in the #45 ?